crypto/ec/ecp_nistp256.c

```diff
@@ -1778,62 +1778,64 @@ struct nistp256_pre_comp_st {
-        ossl_ec_GFp_simple_point_init,
-        ossl_ec_GFp_simple_point_finish,
-        ossl_ec_GFp_simple_point_clear_finish,
-        ossl_ec_GFp_simple_point_copy,
-        ossl_ec_GFp_simple_point_set_to_infinity,
-        ossl_ec_GFp_simple_point_set_affine_coordinates,
-        ossl_ec_GFp_nistp256_point_get_affine_coordinates,
-        0 /* point_set_compressed_coordinates */ ,
-        0 /* point2oct */ ,
-        0 /* oct2point */ ,
-        ossl_ec_GFp_simple_add,
-        ossl_ec_GFp_simple_dbl,
-        ossl_ec_GFp_simple_invert,
-        ossl_ec_GFp_simple_is_at_infinity,
-        ossl_ec_GFp_simple_is_on_curve,
-        ossl_ec_GFp_simple_cmp,
-        ossl_ec_GFp_simple_make_affine,
-        ossl_ec_GFp_simple_points_make_affine,
-        ossl_ec_GFp_nistp256_points_mul,
-        ossl_ec_GFp_nistp256_precompute_mult,
-        ossl_ec_GFp_nistp256_have_precompute_mult,
-        ossl_ec_GFp_nist_field_mul,
-        ossl_ec_GFp_nist_field_sqr,
-        0 /* field_div */ ,
-        ossl_ec_GFp_simple_field_inv,
-        0 /* field_encode */ ,
-        0 /* field_decode */ ,
-        0,                      /* field_set_to_one */
-        ossl_ec_key_simple_priv2oct,
-        ossl_ec_key_simple_oct2priv,
-        0, /* set private */
-        ossl_ec_key_simple_generate_key,
-        ossl_ec_key_simple_check_key,
-        ossl_ec_key_simple_generate_public_key,
-        0, /* keycopy */
-        0, /* keyfinish */
-        ossl_ecdh_simple_compute_key,
-        ossl_ecdsa_simple_sign_setup,
-        ossl_ecdsa_simple_sign_sig,
-        ossl_ecdsa_simple_verify_sig,
-        0, /* field_inverse_mod_ord */
-        0, /* blind_coordinates */
-        0, /* ladder_pre */
-        0, /* ladder_step */
-        0  /* ladder_post */
+        .flags = EC_FLAGS_DEFAULT_OCT,
+        .field_type = NID_X9_62_prime_field,
+        .group_init = ossl_ec_GFp_nistp256_group_init,
+        .group_finish = ossl_ec_GFp_simple_group_finish,
+        .group_clear_finish = ossl_ec_GFp_simple_group_clear_finish,
+        .group_copy = ossl_ec_GFp_nist_group_copy,
+        .group_set_curve = ossl_ec_GFp_nistp256_group_set_curve,
+        .group_get_curve = ossl_ec_GFp_simple_group_get_curve,
+        .grou_get_degree = ossl_ec_GFp_simple_group_get_degree,
+        .group_order_bits gossl_ec_group_simple_order_bits,
+        .group_check_discriminant = ossl_ec_GFp_simple_group_check_discriminant,
+        .point_init = ossl_ec_GFp_simple_point_init,
+        .point_finish = ossl_ec_GFp_simple_point_finish,
+        .point_clear_finish = ossl_ec_GFp_simple_point_clear_finish,
+        .point_copy = ossl_ec_GFp_simple_point_copy,
+        .point_set_to_infinity = ossl_ec_GFp_simple_point_set_to_infinity,
+        .point_set_affine_coordinates =
+            ossl_ec_GFp_simple_point_set_affine_coordinates,
+        .point_get_affine_coordinates =
+            ossl_ec_GFp_nistp256_point_get_affine_coordinates,
+        .point_set_compressed_coordinates,
```

Comment:
missing = NULL?

Reply:
> Tell me you had an editor macro or other tooling to do this :)

I did it by hand, since there weren't that many and they are too inconsistent.